pkg/process/v6/transformers/msrc/transform.go

```diff
@@ -83,18 +83,28 @@ func getFix(vuln unmarshal.MSRCVulnerability) *grypeDB.Fix {
 	return &grypeDB.Fix{
 		Version: fixedInVersion,
 		State:   fixState,
+		Detail:  fixDetail,
 	}
 }
 
 // fixedInKB finds the "latest" patch (KB id) amongst the available microsoft patches and returns it
 // if the "latest" patch cannot be found, an empty string is returned
-func fixedInKB(vulnerability unmarshal.MSRCVulnerability) string {
+func fixedInKB(vulnerability unmarshal.MSRCVulnerability) (string, *grypeDB.FixDetail) {
 	for _, fixedIn := range vulnerability.FixedIn {
 		if fixedIn.IsLatest {
-			return fixedIn.ID
+			var detail *grypeDB.FixDetail
+			if fixedIn.Available.Date != "" {
+				detail = &grypeDB.FixDetail{
+					Available: &grypeDB.FixAvailability{
+						Date: internal.ParseTime(fixedIn.Available.Date),
+						Kind: fixedIn.Available.Kind,
```

Comment:
Does this need to have a non-empty `kind` to be processed correctly downstream?

Reply:
I thought about it but decided that the date information was the primary / most important information here (a missing kind probably shouldn't block a date being conveyed).